File: gemini-analysis/src/main/scala/com/a/eye/gemini/analysis/util/DateUtil.scala
package com.a.eye.gemini.analysis.util

import java.text.SimpleDateFormat
import java.util.Date
import java.util.Calendar
import org.apache.logging.log4j.LogManager
import com.a.eye.gemini.analysis.config.GeminiConfig

object DateUtil {
  private val logger = LogManager.getFormatterLogger(this.getClass.getName)

  private val df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");

  private var partitionTime = new Array[Long](64)

  def date2String(timeStamp: Long): String = {
    df.format(timeStamp)
  }
}

File: gemini-webui/src/main/java/com/a/eye/gemini/webui/util/DateUtil.java
package com.a.eye.gemini.webui.util;

import java.text.SimpleDateFormat;

public class DateUtil {
	private static SimpleDateFormat df = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");

	public static SimpleDateFormat dayDf = new SimpleDateFormat("yyyy/MM/dd");

	public static SimpleDateFormat hourDf = new SimpleDateFormat("HH:mm");

	public static String date2String(Long timeStamp) {
		return df.format(timeStamp);
	}

	public static String data2String(SimpleDateFormat format, Long timeStamp) {
		return format.format(timeStamp);
	}
}
